examples/monaco.html

```diff
@@ -32,9 +32,9 @@
         //// Initialize Firebase.
         //// TODO: replace with your Firebase project configuration.
         var config = {
-            apiKey: "AIzaSyC_JdByNm-E1CAJUkePsr-YJZl7W77oL3g",
```

Comment:
You probably didn't mean to commit your personal config here, can you revert these few lines?

Reply:
Meh, I left it in there because the previous applications also included it. I'm going to be deleting that Firebase instance anyway, so I can delete it or just replace with something like `APIKEY`.